### ISSUE TITLE
Source, which hasn't the lowest priority, is able to control the on/off state of Hyperion NG

### ISSUE THREAD

**Issue by 7h30n3** [hyperion-project/hyperion.ng#1048]
- [x] I confirm that this is an issue rather than a question.

## Bug report
I have Hyperion NG running on an RPi3 (LibreELEC) (priority: 250). On another PC I have Home Assistant 0.116.2 (priority: 128). And on a 3rd PC I use Hyperion Screen Capture (Protocol Buffers Server) to get Ambient Light while gaming (priority: 100).
(With the update to Home Assistant 0.116 I also had to update from Hyperion Classic to NG.)

Home Assistant is controlling Hyperion NG via a bunch of automations.
However when I turn off the light in Home Assistant and start the Screen Capture Software the light keeps off although the Screen Capture Software has lower priority.
If I keep the Screen Capture Software running and turn the light back on via Home Assistant (with a solid color or an effect) the lights also turn on, but with colors according to the caputred screen.

If I remember correctly this didn't used to be the case with Hyperion Classic. There, as soon as a source with a lower priority as Home Assistant was active, settings in Home Assistant where completly ignored.

#### What is expected?
I expected that the source with the lowest priority overwrittes all other settings. 

#### What is actually happening?
Home Assistant has total control over Hyperion NG although it hasn't the lowest priority.

#### System

Hyperion Server: 
- Build:       (HEAD detached at 08909b8a) (Paulchen-Panther-75963ca9/08909b8a-1600115237)
- Build time:  Sep 14 2020 21:04:51
- Git Remote:  https://github.com/hyperion-project/hyperion.ng
- Version:     2.0.0-alpha.8
- UI Lang:     auto (BrowserLang: de)
- UI Access:   default
- Avail Capt:  dispmanx,v4l2,framebuffer,qt

Hyperion Server OS: 
- Distribution: LibreELEC (official): 9.2.3
- Arch:         arm
- Kernel:       linux (4.19.122 (WS: 32))
- Browser:      Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:81.0) Gecko/20100101 Firefox/81.0

**Comment by Joeboyc2** [comment]
I think this is due to the way the new Home Assistant component works, as when you choose to turn off the light in HA, it actually toggles the LED Device switch in the Hyperion Remote Panel.
I don't personally like the way the new config has been added, i had no issues with the previous component working with NG :|

**Comment by 7h30n3** [comment]
Thank you for the clarification. I added this information in the issue I created in the Home Assistant repo: https://github.com/home-assistant/core/issues/41892
I also had no issues with the old hyperion integration.

**Comment by jes1417** [comment]
@Joeboyc2 I had issues with the original component, controlling hyperion.ng did work through automation and scripts but the frontend was broken and the switch in HA would never turn on or off the light entity, it would just toggle back and forth making no changes so it definitely needed to be fixed...

**Comment by Joeboyc2** [comment]
Oh, thats interesting, what version of ng where you running?
I know the first few alphas did not work, but I worked with one of the
developers to get it working with the Home Assistant component and it been
working ever since...

**Comment by jes1417** [comment]
I tried it with alpha 4, 5 and possibly 6 them someone made some changes on a fork of the HA component that I ran as a custom_component and it worked better until the contributor who made the change that made it into core came up with something that I also tested and it worked perfectly but I only use it with a capture card so that may be why it was different for you. I was the one who submitted the original bug report a few months ago in HA you can search it or my username to see the discussions.

**Comment by dermotduffy** [comment]
Hello all! I am the author of the updated Home Assistant Hyperion component (and the new underlying [Python library](https://github.com/dermotduffy/hyperion-py)) -- my changes were inspired by the Hyperion component not working at all for me with Hyperion-ng. I tried to fix it in place, but it was clearly not aimed at the new API (it also had no tests, couldn't use any of the new features such as multiple instances or authentication, nor had a codeowner who was maintaining it).

I am pretty active in working on and trying to improve this component at the moment (e.g. [migrating to config flows](https://github.com/home-assistant/core/pull/42104)) and eager to fix any issues. It is, naturally, working just fine for my use cases -- so I'm keen to learn ways in which it is not working so I can fix them.

Thank you for your patience in this -- I hope Hyperion will be better in Home Assistant in the long-term as a result!

I *think* the issue being described here is because of a choice I made in the implementation of the HA component, and NOT because a change in behavior of Hyperion itself. As such, perhaps we could continue this conversation on the HA bug?
